kit/ChildSession.cpp

```diff
@@ -345,6 +350,17 @@ bool ChildSession::_handleInput(const char *buffer, int length)
 
         return success;
     }
+    else if (tokens.equals(0, "urp")) {
+        if (length < 4) return false;
```

Comment:
Later when you will prepare final PR, please make sure formatting is ok. Here it uses different style than we use in c++ code.
if () {
}
vs
if ()
{
}